mathics/builtin/box/inout.py

```diff
@@ -387,8 +382,6 @@ def apply(self, a, b, evaluation, options):
     def init(self, a, b, options):
         self.box_options = options.copy()
         if not (isinstance(a, BoxConstruct) and isinstance(b, BoxConstruct)):
-            print("a=", a)
```

Comment:
You know if you want to put in print statements and see that actions CI works, you could do this in your own private branch and set that up to run actions. 

Thanks for your consideration.

Reply:
This is why I removed them.

Reply:
I guess what I am saying is that this never should have appeared in a PR, even a draft of a PR.